packages/next/build/webpack/config/blocks/css/index.ts

```diff
@@ -347,6 +347,31 @@ export const css = curry(async function css(
         })
       )
     }
+
+    if (ctx.experimental.appDir) {
+      fns.push(
+        loader({
+          oneOf: [
+            markRemovable({
+              // A global CSS import always has side effects. Webpack will tree
+              // shake the CSS without this option if the issuer claims to have
+              // no side-effects.
+              // See https://github.com/webpack/webpack/issues/6571
+              sideEffects: true,
+              test: regexCssGlobal,
+              issuer: {
+                and: [
+                  ctx.rootDirectory,
+                  /layout(\.client|\.server)?\.(js|mjs|jsx|ts|tsx)$/,
+                ],
+                not: [/node_modules/],
```

Comment:
We want to be able to import css from node_modules for app dir, so I think we don't skip the node_modules dir here?